documentation/ci-fix.md

```diff
@@ -247,35 +247,39 @@ oav validate-spec <openapi-spec-path>
 Please see [readme](https://github.com/Azure/oav/blob/bd04e228b4181c53769ed88e561dec5212e77253/README.md) for how to install or run tool in details.
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
 
-## `Swagger SpellCheck`
-
-If you receive a spelling failure either fix the spelling to match or if there are words that need to be suppressed for your service then add the word to the override list in [cspell.json](https://github.com/Azure/azure-rest-api-specs/blob/main/cSpell.json). Either
-add to your existing section or create a new section for your specific spec or service family if the work is more generally used in lots of files under your service.
-
-``` yaml
- "overrides": [
-    ... example of specific file override
-    {
-        "filename": "**/specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json",
-        "words": [
-            "saskey"
-        ]
-    }
-    ... example of specific service family override
-    {
-        "filename": "**/specification/cognitiveservices/**/*.json",
-        "words": [
-            "flac",
-            "mpga"
-        ]
-    }
-```
+## Spell Check
+
+If you receive a spelling failure, first try to fix the spelling in source.
+
+If there are words that need to be suppressed for your service, add the word to the override list in the `words` list in
+the `specification/<service>/cspell.yaml` file for your service. Specific files can also be overridden using the
+`override` field in your service's `cspell.yaml` file (see example below).
+
+If the `specification/<service>/cspell.yaml` file does not exist, create it using the example below and set the `words`
+and `overrides` fields to the words that need to be suppressed. Both `version` and `import` fields are *required*, use
+the values in the example.
 
-Words are case-insensitive so use lower case for the word list.
+For example (note the words list is sorted alphabetically):
+
+```yaml
+version: '0.2'
```

Comment:
Is version needed?

Reply:
Some basic testing shows that we can get away with not setting it directly in the service's `cspell.yaml` file so we're probably OK to remove it from there. I wouldn't be surprised if a future version of cspell required it or required a different value later, but the version we use today doesn't require it.

Reply:
I leave it to you. It just seems like redundant info we could eliminate.